queries/orderbook/batch_data.sql

```diff
@@ -0,0 +1,46 @@
+WITH batch_data_raw AS (
+    SELECT
+        -- scores
+        concat('0x', encode(ss.winner, 'hex')) as winning_solver,
+        ss.auction_id,
+        ss.block_deadline,
+        ss.winning_score,
+        ss.reference_score,
+        -- settlement
+        CASE WHEN s.tx_hash IS NULL THEN NULL
+            ELSE concat('0x', encode(s.tx_hash, 'hex'))
+        END AS tx_hash,
+        s.block_number,
+        s.log_index,
+        -- settlement observations
+        so.effective_gas_price,
+        so.gas_used,
+        so.surplus,
+        so.fee
```

Comment:
I would suggest that you rename this fee entry in order to more explicitly describe what this is. I know this just references the db, but would make it easier to read if it was something like

`so.fee as total_fee_in_eth`

(if indeed this is what it describes)

Reply:
I will change the name to `total_fee`. Adding the currency fees a bit verbose. We do not write `surplus_in_eth` for example. Maybe we should?

Reply:
I guess you are right that this is probably an overkill, and probably not a great idea, especially if we want to reuse this for Gnosis Chain, for example.